src/System.IO.FileSystem/src/System/IO/WinRTFileSystem.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Please revert the changes to this file.  We don't build or test it with the infrastructure we have right now in the corefx repo, making it difficult to validate changes.